benefits/core/templates/core/includes/sign-out-link.html

```diff
@@ -3,9 +3,13 @@
 
 {% if authentication %}
   {% if authentication.required and authentication.logged_in %}
-    <div class="signout-row">
+    <div class="w-100 position-absolute">
```

Comment:
I saw commit d3f6be0bfba44a7098f674d69c6a77901fd24390 removed the `w-100` class from the previous page link... why do we need it here?

Reply:
Here's a comparison of the Previous page button and the Sign out button:

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/3673236/195219544-a52ca461-0bc6-4ddc-bdcd-cecf63e098ed.png">

(Btw, this is a neat illustration, cos it shows that without changing any CSS and just copying/pasting this Sign Out button bar into the HTML - the buttons behave nicely.)

Because the Previous Page button is aligned horizontally from the left, and HTML goes from left to right, the button is already in the right spot horizontally, without having to apply extra code, like `left: 0`. 

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/3673236/195219805-32277d52-5476-4b03-90ee-245e675340a7.png">

Also, if we do add a `width: 100%` to the Previous Page button parent row, it'll do this:
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/3673236/195219941-2bcf9836-1976-471a-aaa8-b972e762f780.png">

The Sign Out link button though, needs to be positioned to the right-most part of the div. So we need to set a width of 100% so the button knows where to position itself on the right.

This is where the Sign Out link button would position itself if there wasnt the `w-100` class (even if you add `width: 100% to the rest of the children):

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/3673236/195220131-cb1ddc05-ea11-4000-b90d-8d1268fc714b.png">

Reply:
🤯 Thank you for this detailed explanation!!

Please write those CSS docs you were talking about and include this kind of stuff! 😍